### PR TITLE
feat: update 2 out of 3 lessons with POKT rpc providers

### PR DESCRIPTION
**Pull request by elPiablo**
## Changed

- [x] updated apps/academy/src/pages/tracks/nft-solidity/2.mdx as tastefully as possible - there is no explicit reference to alternative providers, but there **is** still a reference to Ankr in the `hardhat.config` file.
- [x] apps/academy/src/pages/tracks/nft-solidity/3.mdx - there is no mention of alternative RPC node providers

## Unable to change (see issue below)
- apps/academy/src/pages/tracks/nft-solidity/5.mdx imports `alchemyProvider` for `WAGMI`. Is it possible to change this?
I created this issue to address it: https://github.com/Developer-DAO/academy-turbo/issues/118

this is ready for review. Don't hold back on your suggestions to change the texts to something else, or change them back in some way. I'm just iterating on trying to get a good balance that isn't shilling too much. I appreciate your insights and wisdom 😁

### CODE REVIEW

File: apps/academy/src/pages/tracks/nft-solidity/2.mdx [hunk 1]
```diff
@@ -927,7 +925,7 @@ require("@nomicfoundation/hardhat-toolbox");
 
 const WALLET_PRIVATE_KEY = "YOUR-PRIVATE-KEY-DONT-SHARE";
 
-const RPC_API_KEY = "YOUR-API-KEY-FROM-INFURA-OR-ALCHEMY";
+const RPC_API_KEY = "YOUR-API-KEY-FROM-YOUR-RPC-PROVIDER";
```
Comment (Markkos89) [inline apps/academy/src/pages/tracks/nft-solidity/2.mdx:928]: Here I blieve that you don't use the API_KEY rather you use the actual RPC_URL as said above in the JSON object:

```
sepolia: {
   url: RPC_URL,
   ...
   }
```

**Review by Markkos89 — CHANGES_REQUESTED** [review]

File: apps/academy/src/pages/tracks/nft-solidity/2.mdx (hunk 1 above)
Comment (elPiablo) [reply apps/academy/src/pages/tracks/nft-solidity/2.mdx:928]: what should we change it to @Markkos89 ?

Comment (Markkos89) [reply apps/academy/src/pages/tracks/nft-solidity/2.mdx:928]: `RPC_URL` as this is an url starting with `https://..`

Comment (elPiablo) [reply apps/academy/src/pages/tracks/nft-solidity/2.mdx:928]: "YOUR-API-KEY-URL-FROM-YOUR-RPC-PROVIDER"?

Comment (elPiablo) [reply apps/academy/src/pages/tracks/nft-solidity/2.mdx:928]: "YOUR-API-KEY-URL-FROM-RPC-PROVIDER"

Comment (elPiablo) [reply apps/academy/src/pages/tracks/nft-solidity/2.mdx:928]: if it is the first time a learner is using these tools, they won't see them very easily - that is the reason that it is written with a lot of words - to easily attract their attention to it.

Comment (Markkos89) [reply apps/academy/src/pages/tracks/nft-solidity/2.mdx:928]: it would be `YOUR-RPC-URL-FROM-YOUR-RPC-PROVIDER` because the string is the RPC URL

**Review by Markkos89 — APPROVED** [review]

**Comment by elPiablo** [comment]
Cheers @Markkos89 !!

**Pull request merged**